ajax/libs/SoundJS/package.json

```diff
@@ -1,23 +1,31 @@
 {
   "name": "SoundJS",
+  "npmName": "soundjs",
   "filename": "soundjs.min.js",
-  "version": "0.6.0",
-  "description": "A Javascript library that provides a simple API, and powerful features to make working with audio a breeze.",
-  "homepage": "http://www.createjs.com/#!/SoundJS",
-  "keywords": [
-    "sound",
-    "audio",
-    "webAudio"
+  "description": "A JavaScript library that provides a simple API, and powerful features to make working with audio a breeze. Easily ties in audio file loading to PreloadJS.",
+  "npmFileMap": [
```

Comment:
Let's use git auto-update instead because there are more versions on GitHub.
You can set up it by removing `npmName` and `npmFileMap` then add
```javascript
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/CreateJS/SoundJS.git",
    "fileMap": [
      {
        "basePath": "lib",
        "files": [
          "+(*plugin|soundjs)*"
        ]
      }
    ]
  }
```

Here I suggest we use `+(*plugin|soundjs)*` as the pattern for auto-update config because we want to prevent unnecessary files such as `README.txt` from being fetched.

Reply:
@extend1994 The contributor is one of member of CreateJS(https://github.com/orgs/CreateJS/people)
Maybe we need to discuss with him that which auto-update does he want, npm or git.
Also, I think some field's order needs to be followed current package.json.

What do you think?

Similar to #11852 #11851

Reply:
He could let us know if he don't want git as the source. Also we can wait him for the response in #11851.